src/backend/api/reservations.js

```diff
@@ -0,0 +1,58 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+router.get("/", async (request, response) => {
+    try {
+        const reservations = await knex("reservation");
+        response.json(reservations);
+    } catch (error) {
+        throw error;
+    }
+});
```

Comment:
In JS, we normally only want to indent with 2 spaces; here it seems like 4 spaces are used.